packages/yew-macro/tests/html_lints/fail.rs

```diff
@@ -0,0 +1,14 @@
+use yew::prelude::*;
+
+fn main() {
+    let bad_a = html! {
+        <a>{"I don't have a href attribute"}</a>
```

Comment:
Nit: I think the official style used by Yew has spaces between the `{` and the quotes (e.g. `<a>{ "I don't have a href attribute" }</a>`. Although https://github.com/yewstack/yew/pull/1774 would make this redundant.